lib/containers/utils.pm

```diff
@@ -41,20 +43,65 @@ sub test_seccomp {
     }
 }
 
-sub basic_container_tests {
-    my %args    = @_;
-    my $runtime = $args{runtime};
-    die "You must define the runtime!" unless $runtime;
+sub check_docker_firewall {
+    my $container_name = 'sut_container';
+    my $docker_version = get_docker_version();
+    systemctl('is-active firewalld');
+    my $running = script_output qq(docker ps -q | wc -l);
+    validate_script_output('ip a s docker0', sub { /state DOWN/ }) if $running == 0;
+    assert_script_run "firewall-cmd --list-all --zone=docker";
+    # Docker zone is created only for SLE15-SP3 for version <20.10
+    validate_script_output "firewall-cmd --list-interfaces --zone=docker", sub { /docker0/ } if (check_runtime_version($docker_version, ">=20.10") || is_sle("15-SP3+"));
+    validate_script_output "firewall-cmd --list-interfaces --zone=trusted", sub { /^\s*$/ };
```

Comment:
What this line does?

Reply:
Same thing asks @asmorodskyi . https://docs.docker.com/network/iptables/#integration-with-firewalld. just another check

Reply:
I found this [question](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12480#discussion_r627278432) from Anton:
> what this line suppose to prove ? that some interfaces are in trusted ? how it is related to docker + firewalld ?

I read your link but I still have the same question :)

Reply:
maybe i am wrong but Seems that when firewalld was used in some cases you need to have the docker0 in trusted zone to have internet access. based on the documentation and the default setup i try to ensure that docker0 is not set for that zone. i could also check all the zones but i thought it was not necessary for the others

Reply:
by default? not if not someone added IMO.